include/eve/module/core/regular/bit_ternary.hpp

```diff
@@ -295,19 +302,19 @@ namespace eve
       if constexpr(K == 0xa9) return bit_not(bit_xor(c, bit_or(b, a)));
       if constexpr(K == 0xaa) return c;
       if constexpr(K == 0xab) return bit_or(c, bit_not(bit_or(b, a)));
-      if constexpr(K == 0xac) return bit_select(a, c, b);
+      if constexpr(K == 0xac) return bit_select(a, c, b);//bit_or(bit_and(c, a), bit_andnot(b, a));
       if constexpr(K == 0xad) return bit_or(bit_and(b, c), bit_xor(a, bit_not(c)));
       if constexpr(K == 0xae) return bit_or(c, bit_notand(a, b));
       if constexpr(K == 0xaf) return bit_ornot(c, a);
       if constexpr(K == 0xb0) return bit_notand(bit_notand(c, b), a);
-      if constexpr(K == 0xb1) return bit_select(c, a, bit_not(b));
-      if constexpr(K == 0xb2) return bit_select(b, bit_and(a, c), bit_or(a, c));
+      if constexpr(K == 0xb1) return bit_or(bit_and(a, c), bit_andnot(bit_not(b), c));
+      if constexpr(K == 0xb2) return bit_or(bit_and(b, a, c),bit_notand(b, bit_or(a, c)));
```

Comment:
Why are these changing? Was the previous implementation buggy? Or is it just better?

Reply:
buggy